teamA/jinwoo/controllers/postController.js

```diff
@@ -0,0 +1,51 @@
+const postService = require('../services/postService');
+
+const postUp = async (req, res) => {
+  try {
+    const { title, content, userId } = req.body;
+
+    if ( !title || !content || !userId ) {
+      return res.status(400).json({ message : 'KEY_ERROR' });
+    }
+
+    await postService.postUp( title, content, userId );
+    return res.status(201).json({ message : 'POSTUP_SUCCESS' });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message : err.message });
+  }
+};
+
+const postList = async (req, res) => {
+    const postsWesta = await postService.postList();
```

Comment:
```suggestion
const getPostList = async (req, res) => {
    const postsWesta = await postService.getPostList();
```
액션 + 대상으로 네이밍 해주세요~~